src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -362,9 +362,12 @@ export class ExtensionsViewlet extends ViewContainerViewlet implements IExtensio
 		this.searchBox.setModel(this.modelService.createModel('', null, uri.parse('extensions:searchinput'), true));
 
 		this.disposables.push(this.searchBox.onDidPaste(() => {
-			this.searchBox.setValue(this.searchBox.getValue().replace(/\s+/g, ' '));
+			let trimmed = this.searchBox.getValue().replace(/\s+/g, ' ');
+			this.searchBox.setValue(trimmed);
 			this.searchBox.setScrollTop(0);
```

Comment:
why do we need a scroll top if we ever only expect this to be a single line editor?

Reply:
A weird shadow appears above the editor when you try to past multiline text otherwise. I think its becuase we technically paste the multiline, then change it to single line, so internally it thinks its multiline for a bit and tries to render the shadow to show there's more above?